### PR TITLE
more complete explanation of npm based use

### PR DESCRIPTION
**Pull request by dmison**
As per https://github.com/twbs/bootstrap/issues/19067

### CODE REVIEW

**Comment by StudyZhou** [comment]
Hello, this is just a test.

File: docs/_includes/getting-started/download.html [hunk 1]
```diff
@@ -47,7 +47,12 @@ <h2 id="download-bower">Install with Bower</h2>
   <h2 id="download-npm">Install with npm</h2>
   <p>You can also install Bootstrap using <a href="https://www.npmjs.com">npm</a>:</p>
   {% highlight bash %}$ npm install bootstrap@3{% endhighlight %}
-  <p><code>require('bootstrap')</code> will load all of Bootstrap's jQuery plugins onto the jQuery object. The <code>bootstrap</code> module itself does not export anything. You can manually load Bootstrap's jQuery plugins individually by loading the <code>/js/*.js</code> files under the package's top-level directory.</p>
```
Comment (StudyZhou) [inline docs/_includes/getting-started/download.html]: Hello, this is just a test.

Comment (patrickhlauke) [inline docs/_includes/getting-started/download.html]: don't use public repositories to test how to use github

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team

**Pull request closed without merging**